test/test.js

```diff
@@ -2864,6 +2864,16 @@ exports['wrap EventEmitter (or jQuery) on handler'] = function (test) {
     });
 };
 
+exports['removing EventEmitter (or jQuery) listener on destruction'] = function(test) {
```

Comment:
Sorry, I meant in the test name. LIke [this](https://github.com/caolan/highland/blob/master/test/test.js#L476).